pulse2percept/api.py

```diff
@@ -54,25 +51,82 @@ def __init__(self, implant, engine='joblib', scheduler='threading',
+                              datapath='.', save_data=True):
         """Sets parameters of the optic fiber layer (OFL)
 
         Parameters
         ----------
         sampling : float, optional, default: 100 microns
             Microns per grid cell.
-        axon_lambda : float, optional, default: 2
-            Constant that determines fall-off with axonal distance.
-        x_range : list|None, default: None
+        x_range : (xlo, xhi)|None, default: None
             Lower and upper bound of the retinal grid (microns) in horizontal
-            dimension. Either a list [xlo, xhi] or None. If None, the generated
-            grid will be just big enough to fit the implant.
-        y_range : list|None, default: None
+            dimension. Either a tuple (xlo, xhi) or None. If None, the
+            generated grid will be just big enough to fit the implant.
+        y_range : (ylo, yhi)|None, default: None
             Lower and upper bound of the retinal grid (microns) in vertical
-            dimension. Either a list [ylo, yhi] or None. If None, the generated
-            grid will be just big enough to fit the implant.
+            dimension. Either a tuple (ylo, yhi) or None. If None, the
+            generated grid will be just big enough to fit the implant.
+        n_axons : int, optional, default: 501
+            The number of axons to generate. Their start orientations `phi0`
+            (in modified polar coordinates) will be sampled uniformly from
+            `phi_range`.
+        phi_range : (lophi, hiphi), optional, default: (-180, 180)
+            Range of angular positions of axon fibers at their starting points
+            (polar coordinates, degrees) to be sampled uniformly with `n_axons`
+            samples. Must be within [-180, 180].
+        n_rho: int, optional, default: 801
+            Number of sampling points along the radial axis(polar coordinates).
+        rho_range: (rho_min, rho_max), optional, default: (4.0, 45.0)
+            Lower and upper bounds for the radial position values(polar
+            coordinates).
+        loc_od: (x_od, y_od), optional, default: (15.0, 2.0)
+            Location of the center of the optic disc(x, y) in Cartesian
+            coordinates.
+        sensitivity_rule : {'decay', 'Jeng2011'}, optional, default: 'decay'
+            This rule specifies how the activation of the axon differs as a
+            function of distance from the soma. The following options are
+            available:
+            - 'decay':
+                Axon sensitivity decays exponentially with distance. Specify
+                `decay_const` to change the steepness of the fall-off with
+                distance.
+            - 'Jeng2011':
+                Axon sensitivity peaks near the sodium band (50um from the
+                soma), then plateaus on the distal axon at roughly half of the
+                peak sensitivity. See Figure 2 in Jeng, Tang, Molnar, Desai,
```

Comment:
See https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt on how to integrate references in docstrings,.